docs/data/tree-view/tree-item-customization/useTreeItemHookPublicAPI.tsx

```diff
@@ -34,9 +41,21 @@ const CustomTreeItem = React.forwardRef(function CustomTreeItem(
   props: TreeItemProps,
   ref: React.Ref<HTMLLIElement>,
 ) {
-  const { publicAPI } = useTreeItem(props);
+  const { publicAPI, status } = useTreeItem(props);
 
-  const childrenNumber = publicAPI.getItemOrderedChildrenIds(props.itemId).length;
+  const selectFirstChildren = status.expanded
```

Comment:
The old example was no longer doable.
For now `useSelector` and all the selectors are not public (unlike on the grid), I'd prefer to wait for some user feedbacks and requests.

@noraleonte if you have a better of something we could demonstrate that do not require a `publicAPI` usage in the render